06-unit-testing-and-mocking/lab/solution/src/bg/sofia/uni/fmi/mjt/shopping/ItemNotFoundException.java

```diff
@@ -0,0 +1,8 @@
+package bg.sofia.uni.fmi.mjt.shopping;
+
+public class ItemNotFoundException extends RuntimeException {
+
```

Comment:
Remove the empty line, or add an empty line after the constructor, to make it symmetrical.